UI/DialogSystem/DialogSystem.cs

```diff
@@ -0,0 +1,71 @@
+using Godot;
+using System;
+using System.Collections.Generic;
+using System.Threading.Tasks;
+
+public class DialogSystem : CanvasLayer 
+{
+    private RichTextLabel _textLabel;
+    private IEnumerator<float> _coRoutine;
+    private Queue<string> _dialogQueue = new Queue<string>();
+
```

Comment:
can you remove those extra empty lines? they're not necessary and it just makes files bigger and less easy to read

Reply:
Indeed (Another good reason to get autolinting up and running :D)